doc/api/deprecations.md

```diff
@@ -630,6 +630,14 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP0073"></a>
+### DEP0073: net.Socket.prototype.listen()
```

Comment:
The deprecation code here should be `DEP00XX`. The actual number should be assigned when the PR lands.